### ISSUE TITLE
Everything is working but the mic isn't transcribing properly

### ISSUE THREAD

**Issue by acophisoph** [AlizerUncaged/desktop-waifu#85]
When I press Ctrl to activate the mic I get this message
You (mic [Transcribing (44 Bytes)]) > Error: <empty message>

**Comment by acophisoph** [comment]
Also for anyone who is reading this installing node.js might help with some of the issues you are having. I know I had to install node.js to get the Node files running

**Comment by JackRobert94464** [comment]
at this point im just not bothering with the mic but modify the code to use direct input from console lol

**Comment by acophisoph** [comment]
@JackRobert94464 yeah thats a good idea! Do you know where the code is to modify the audio to take direct input from console? I was looking at transcribe.py, audio.py but couldn't find it for some reason

**Comment by JackRobert94464** [comment]
it's a pretty simple and rugged fix actually... Just commented out all the transcribing stuff and replace the input for message as input() function like so
![image](https://github.com/AlizerUncaged/desktop-waifu/assets/100371269/8b2ba781-3333-4c50-98b6-994b7cb7d956)
You still have to do left ctrl - left shift though, but transcribing should not be working and instead you are prompted for typing into the console.

**Comment by Ivonchixd** [comment]
With that code you can actually fix the error when transcribing audio? (Im getting: File "F:\Programs\AI\desktop-waifu-master\waifu\Src\runner.py", line 147, in
utils.characterAi.send_message_to_process_via_websocket(transcript)
File "F:\Programs\AI\desktop-waifu-master\waifu\Src\utils\characterAi.py", line 31, in send_message_to_process_via_websocket
ws.send(message)
AttributeError: 'NoneType' object has no attribute 'send')

> it's a pretty simple and rugged fix actually... Just commented out all the transcribing stuff and replace the input for message as input() function like so ![image](https://user-images.githubusercontent.com/100371269/247395600-8b2ba781-3333-4c50-98b6-994b7cb7d956.png) You still have to do left ctrl - left shift though, but transcribing should not be working and instead you are prompted for typing into the console.

**Comment by JackRobert94464** [comment]
@Ivonchixd It's not a "fix" but rather changing the input method completely (mostly for debug or if u just want to get ur waifu up and running as fast as possible without much code experience). Someone here already have an alternative solution of using a local whisper server, you may want to check that out 🙂
